Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -127,6 +127,55 @@ public function importResourcesCommand($simulate = false)
     }
 
     /**
+     * Remove unused assets
+     *
+     * This command iterates over all existing assets, checks their usage count
+     * and lists the assets which are not reported as used by any AssetUsageStrategies.
+     * The unused assets can than be removed.
+     */
+    public function removeUnusedCommand()
+    {
+        $iterator = $this->assetRepository->findAllIterator();
+        $assetCount = $this->assetRepository->countAll();
+        $unusedAssetInfo = [];
+        $unusedAssetCount = 0;
+
+        $this->outputLine('<b>Searching for unused assets.</b>');
+
+        $this->output->progressStart($assetCount);
+        /** @var Asset $asset */
+        foreach ($this->assetRepository->iterate($iterator) as $asset) {
+            if ($asset->getUsageCount() === 0) {
+                $unusedAssetInfo[] = sprintf('- Found unused asset %s with file name %s', $asset->getIdentifier(), $asset->getResource()->getFilename());
```

Comment:
Instead of outputting this for each asset, maybe just output "%s with file name %s".

Reply:
Oh yes!